app/scripts/services/shims/hawtioExtension.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+angular
+ .module('openshiftConsole')
+ .factory('HawtioExtension', [
+   'extensionRegistry',
+   function(extensionRegistry) {
+     console.warn('HawtioExtension.add() has been deprecated.  Please migrate to angular-extension-registry https://github.com/openshift/angular-extension-registry');
+     return {
+       add: function(endpointName, domNodeGenerator) {
+         extensionRegistry.add(endpointName, function(args) {
+           return {
+             type: 'dom',
+             node: domNodeGenerator(args)
+           }
+         });
+       }
```

Comment:
missing `;`

Reply:
@spadgett  I tested with some of that code, yes.  I just swapped to points like `'nav-help-dropdown'`.

Reply:
`;` is our `jshint` job not running?  I don't get any errors locally..